src/components/FileRepoSidebar/FileManifestsDownloadInput.js

```diff
@@ -10,28 +10,21 @@ import { ActionButton } from 'uikit/Button';
 import FamilyManifestModal from '../FamilyManifestModal';
 
 export default compose(injectState, withTheme)(({ theme, effects: { setModal }, ...props }) => (
-  <div
-    css={`
-      display: flex;
-      margin-bottom: 13px;
-    `}
+  <ActionButton
+    onClick={() =>
+      setModal({
+        title: 'Download Manifest',
+        component: <FamilyManifestModal {...props} />,
+      })
+    }
   >
-    <ActionButton
-      onClick={() =>
-        setModal({
-          title: 'Download Manifest',
-          component: <FamilyManifestModal {...props} />,
-        })
-      }
-    >
-      <DownloadIcon
-        className={css`
-          margin-right: 9px;
-        `}
-      />
-      <span css={theme.uppercase}>
-        <Trans>Download</Trans>
-      </span>
-    </ActionButton>
-  </div>
+    <DownloadIcon
+      className={css`
+        margin-right: 9px;
+      `}
```

Comment:
this is where I start to think having the styled-system margin / padding stuff may be super clean.

not for now but maybe refactor phase 2 lol